docs/getting-started/components/registry.md

```diff
@@ -1,31 +1,47 @@
 # Registry
 
-The Feast feature registry is a central catalog of all the feature definitions and their related metadata. It allows data scientists to search, discover, and collaborate on new features.
+Feast uses a registry to store all applied Feast objects (e.g. Feature views, entities, etc). It allows data scientists to search, discover, and collaborate on new features. The registry exposes methods to apply, list, retrieve and delete these objects, and is an abstraction with multiple implementations.
```

Comment:
I like this too so maybe just a modification.

Reply:
Sure, I'll merge the two.